test/jdk/java/nio/channels/DatagramChannel/SendReceiveMaxSize.java

```diff
@@ -103,16 +105,31 @@ public Object[][] invariants() throws IOException {
                     .filter(Predicate.not(InetAddress::isLoopbackAddress))
                     .findFirst()
                     .orElse((Inet6Address) InetAddress.getByName("::1"));
-            testcases.add(new Object[]{
-                    supplier(() -> DatagramChannel.open()),
-                    IPV6_SNDBUF,
-                    IPv6Addr
-            });
-            testcases.add(new Object[]{
-                    supplier(() -> DatagramChannel.open(INET6)),
-                    IPV6_SNDBUF,
-                    IPv6Addr
-            });
+            if (Platform.isAix()) {
+                testcases.add(new Object[]{
+                        supplier(() -> DatagramChannel.open()),
+                        IPV6_SNDBUF_AIX,
+                        IPv6Addr
+                });
+                testcases.add(new Object[]{
+                        supplier(() -> DatagramChannel.open(INET6)),
+                        IPV6_SNDBUF_AIX,
+                        IPv6Addr
+                });
+            }
+            else
```

Comment:
Since the only thing changing here is the buffer size value passed to `testcases.add` then maybe using a local variable could help reduce duplication. Something like:

```
int maxDatagramSize = Platform.isAix() ? IPV6_SNDBUF_AIX : IPV6_SNDBUF;
```

As Alan suggested this could better be placed in e.g. `IPSupport`.